twopane-navigation/src/utilities/constants.ts

```diff
@@ -0,0 +1 @@
+export const hingeWidth: number = 33.6;
```

Comment:
We should avoid using hard-coded sizes.  Can  this use the hingeWidth property from dualscreeninfo?  This queries the actual device.

Reply:
very smart, I completely spaced on the dualscreeninfo.hingeWidth property to be honest. adding this now